client/my-sites/migrate/section-migrate.jsx

```diff
@@ -319,26 +321,26 @@ class SectionMigrate extends Component {
 					align="center"
 				/>
 				<div className="migrate__status">
-					There was an error with your import.
+					{ translate( 'There was an error with your import.' ) }
 					<br />
 					{ this.state.errorMessage }
 				</div>
 				<Button primary onClick={ this.resetMigration }>
-					Back to your site
+					{ translate( 'Back to your site' ) }
 				</Button>
 			</Card>
 		);
 	}
 
 	renderMigrationProgress() {
-		const { sourceSite, targetSite } = this.props;
+		const { sourceSite, targetSite, translate } = this.props;
 		const sourceSiteDomain = get( sourceSite, 'domain' );
 		const targetSiteDomain = get( targetSite, 'domain' );
 		const subHeaderText = (
 			<>
-				{ "We're moving everything from " }
+				{ translate( "We're moving everything from " ) }
```

Comment:
This won't translate well. Sentences like this should use [string substitution](https://github.com/Automattic/i18n-calypso#string-substitution).

Reply:
Thanks! Updated with proper string and component substitution!

Reply:
I wonder if Jon meant having placeholders for the `sourceSiteDomain` and `targetSiteDomain` and their wrapping elements? I think that is possible.